### PR TITLE
Win dependencies 2

### PR DESCRIPTION
**Pull request by rasapala**
### 🛠 Summary

JIRA CVS-157903
## Install dependencies for ovms windows build on CI or on laptop.
open cmd.exe and run windows_install_dependencies.bat with target directory (default drive c:\\opt) and force reinstall flag 0/1
```
windows_install_dependencies.bat
```

### 🧪 Checklist

- [ ] Unit tests added.
- [x] The documentation updated.
- [ ] Change follows security best practices.
``

### CODE REVIEW

File: docs/windows_developer_guide.md [hunk 1]
```diff
@@ -89,37 +77,18 @@ set HTTP_PROXY=my.proxy.com:123
...
-
 ## Run Developer Command Prompt for VS 2019
-## Enable Developer mode on in windows system settings
-
-#### Boring SSL - not needed until md5 hash is needed.
-Clone in in c:\opt\
-```
-git clone https://github.com/firebase/boringSSL-SwiftPM.git
-```
+Enable Developer mode on in windows system settings
```
Comment (mzegla) [inline docs/windows_developer_guide.md:81]:
```suggestion
Enable Developer mode in windows system settings
```

File: docs/windows_developer_guide.md [hunk 2]
```diff
@@ -89,37 +77,18 @@ set HTTP_PROXY=my.proxy.com:123
...
 
-# Opencl headers
+## Install dependencies
+open cmd.exe and run windows_install_dependencies.bat with target directory (default drive c:\\opt) and force reinstall flag 0/1
```
Comment (mzegla) [inline docs/windows_developer_guide.md:84]: What does the second part mean? `with target directory (default drive c:\\opt) and force reinstall flag 0/1`
How does it relate to below line?

File: windows_install_dependencies.bat [hunk 3]
```diff
@@ -13,24 +13,54 @@
...
+  set "in_path=%1"
+)
+if "%2"=="" (
+  echo No argument provided. Using default force = 0
+  set "force=0"
+) else (
+  echo Argument provided: Using force = %2
+  set "force=1"
+)
+set "BAZEL_SHORT_PATH=C:\%in_path%"
```
Comment (mzegla) [inline windows_install_dependencies.bat:37]: Maybe let's use names as described in parameters?
`in_path -> output_user_root`
`force -> expunge`

File: windows_install_dependencies.bat [hunk 4]
```diff
@@ -13,24 +13,54 @@
...
 
 :: Set proper PATH environment variable: Remove other python paths and add c:\opt with bazel, wget to PATH
 set "PATH=%setPath%"
 
-:: OpenVINO
+:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+::::::::::::::::::::::: Install in c:\PR-1234\ section started - once per build, reinstalled only with expunge clean ::::::::::::::::::::::::::::::::::
```
Comment (mzegla) [inline windows_install_dependencies.bat:44]: PR-1234 sounds a bit too "real". 
Can we have something like `PR-XXXX` so it's obvious we don't mean hardcoded path here?

File: windows_opencv.bat [hunk 5]
```diff
@@ -0,0 +1,77 @@
+::
```
Comment (mzegla) [inline windows_opencv.bat:1]: Maybe we could group all Windows installation scripts in a separate directory?

**Review by mzegla — APPROVED** [review]

**Review by dkalinowski — APPROVED** [review]

**Pull request merged**